CybridSDKTests/Components/Transfer/View/TransferViewModelTest.swift

```diff
@@ -87,7 +88,24 @@ class TransferViewModelTest: XCTestCase {
         XCTAssertFalse(viewModel.externalBankAccounts.value.isEmpty)
         XCTAssertEqual(viewModel.uiState.value, .ACCOUNTS)
         XCTAssertEqual(viewModel.modalUIState.value, .LOADING)
-        XCTAssertEqual(viewModel.balanceLoading.value, .CONTENT)
+    }
+
+    func test_fetchExternalAccounts_Successfully_Empty() {
+
+        // -- Given
+        let viewModel = createViewModel()
+
+        // -- When
+        viewModel.modalUIState.value = .LOADING
+        dataProvider.fetchExternalBankAccountsSuccessfully_Empty()
+        viewModel.fetchExternalAccounts()
+        dataProvider.fetchExternalBankAccountsSuccessfully_Empty()
+
+        // -- Then
+        XCTAssertNotNil(viewModel)
+        XCTAssertTrue(viewModel.externalBankAccounts.value.isEmpty)
+        XCTAssertEqual(viewModel.uiState.value, .WARNING)
+        XCTAssertEqual(viewModel.modalUIState.value, .LOADING)
     }
```

Comment:
We don't have tests to ensure that when a bank account needs to be refreshed we show the correct information?